Poke/App.tsx

```diff
@@ -94,9 +94,9 @@ const App = (props: NavigationComponentProps) => {
             ) : (
               <>
                 <Text style={styles.simple}>Bem vindo(a) à Taqtile!</Text>
-                <EmailInput text={email} onTextChange={setEmail} />
-                <PasswordInput text={password} onTextChange={setPassword} />
-                <SubmitButton onTap={handleSubmit} />
+                <Input name={'E-mail'} text={email} onTextChange={setEmail} />
+                <PasswordInput name={''} text={password} onTextChange={setPassword} />
+                <SubmitButton text={'Entrar'} onTap={handleSubmit} />
```

Comment:
Props que são strings dá pra omitir as chaves:
```suggestion
                <Input name='E-mail' text={email} onTextChange={setEmail} />
                <PasswordInput name='' text={password} onTextChange={setPassword} />
                <SubmitButton text='Entrar' onTap={handleSubmit} />
```